Taskfile.yml

```diff
@@ -0,0 +1,122 @@
+deps:
+  desc: Install all dependencies
+  cmds:
+   - task: deps:be
+   - task: deps:fe
+   - task: deps:watch
+
+deps:be:
```

Comment:
`be/fe` = `backend/frontend`? Never seen this abbreviation before. :)

Reply:
yeah exactly, we always use these abbreviations, but if you feel it is unclear i'm happy to change it

Reply:
nop, I'm ok with it :)